bluemira/equilibria/flux_surfaces.py

```diff
@@ -123,6 +123,43 @@ def connection_length(self, eq):
         """
         return np.sum(self._dl(eq))
 
+    def poloidal_angle(self, eq, x_strike, z_strike, gamma):
+        """
+        From glancing angle to poloidal angle.
+
```

Comment:
1. Simpler formula with sin to be used
2. Please write the formula in LaTeX here